DefaultSettings.php

```diff
@@ -697,15 +697,23 @@
 	##
 
 	###
+	# Change propagation watchlist
+	#
 	# Properties (usually given as internal ids or DB key versions of property
 	# titles) that are relevant for declaring the behavior of a property P on a
 	# property page in the sense that changing their values requires that all
-	# pages that use P must be processed again. For example, if _PVAL (allowed
-	# values) for a property change, then pages must be processed again. This
-	# setting is not normally changed by users but by extensions that add new
-	# types that have their own additional declaration properties.
+	# pages that use P must be processed again.
+	#
+	# For example, if _PVAL (allowed values) for a property change, then pages
+	# must be processed again. This setting is not normally changed by users but
+	# by extensions that add new types that have their own additional declaration
+	# properties.
+	#
+	# @since 1.6
```

Comment:
@mwjames This setting was actually introduced with 1.5.0

Reply:
... one this one will be renamed to reflect that it is actually a `smwgChangePropagationWatchlist`